python/py-graphene/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        graphql-python graphene 3.0.0 v
+github.setup        graphql-python graphene 3.1.0 v
```

Comment:
latest version is 3.1.1 - can we update to that ; otherwise looks good to me and we can merge quickly